src/execution/futex-emulation.cc

```diff
@@ -105,11 +105,22 @@ namespace {
 // and `interrupted_` fields for each individual list node that is currently
 // part of the list. It must be the mutex used together with the `cond_`
 // condition variable of such nodes.
-base::LazyMutex g_mutex = LAZY_MUTEX_INITIALIZER;
+struct RecordReplayOrderedFutexEmulationMutexConstructor {
+  // Constructs the mutex with an ordered name.
+  static void Construct(void* allocated_ptr) { new (allocated_ptr) base::Mutex("FutexEmulation"); }
+};
+using LazyMutex =
+  base::LazyStaticInstance<
```

Comment:
I'm not sure how this works and I can't find it in the code - I guess you researched this, so 👍 from me 😅